internal/moreos/moreos_test.go

```diff
@@ -140,7 +142,7 @@ func TestProcessGroupAttr_Interrupt(t *testing.T) {
 
 func Test_EnsureProcessDone(t *testing.T) {
 	// Fork a process that hangs
-	cmd := exec.Command("cat" + Exe)
+	cmd := exec.Command("sleep"+Exe, "1000")
```

Comment:
When using "cat", which seems "blocking"/"hanging", calling `cmd.Wait` does not block the main thread (hence it is not validating the value of "killing" the process).

Reply:
please roll this back. this is only testing that calling interrupt causes the process to complete and that calling it when a process is no longer there doesn't panic.

The cmd.Wait is commented as having undefined (platform specific) output. That's why it is not checked, and that is ok. The main thing is reduce flakes and this test hasn't flaked on any of our operating systems.

Let's please not change this variable because it confuses the topics, ex is a change here causing a regression elsewhere. ITOW roll back the replacement of cat with sleep and also rollback the timeout override as our tests have been very stable. If something is now not stable we should have an easier time understanding why as it would be a change here.